### PR TITLE
Remove app from docker compose to speedup development

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Remove app from docker compose to speedup development

**Why**
Remove app from docker compose to speedup development

**Added feature flags**

**Affected issues**  

**How has this been tested?**  

**Whats Next?**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,4 +20,4 @@ RUN yarn install --frozen-lockfile
 #   https://github.com/nodejs/node/issues/16336#issuecomment-568845447
 ENV NODE_EXTRA_CA_CERTS=node_modules/node_extra_ca_certs_mozilla_bundle/ca_bundle/ca_intermediate_root_bundle.pem
 
-CMD ["yarn", "start"]
+CMD ["yarn", "start:no-spa"]
```
Comment (bgvozdev) [inline Dockerfile:23]: Do we still need this Dockerfile?

Comment (gxueatlassian) [reply Dockerfile:23]: e2e test, haha

**Review by bgvozdev — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -88,12 +88,12 @@ The first time you run the app, simply run:
 ```
 yarn install # installs node modules
 docker-compose up # Spin up docker containers
+yarn start #Spin up websever and worker
 ```
 
-That's it.  Everything is ran in docker-compose, including redis, postgres, ngrok and the app (main and worker thread).
-If you want to debug, you can connect to the remote port of 9229 for the main thread and 9230 for the worker thread in docker.  Any changes to the code will restart the node server automatically.
+That's it.  Dependant services ran in docker-compose, including redis, postgres, ngrok and the app (main and worker thread) run in local `yarn start`.
```
Comment (krazziekay) [inline CONTRIBUTING.md:94]: This is a bit confusing cause **redis, postgres, ngrok and the app** kind of imply that the **app** is also one of the dependant services, can we reword this sentence?
```suggestion
That's it.  Dependant services ran in docker-compose, including redis, postgres and ngrok. And now to run the app (main and worker thread), please run in local `yarn start`.
```

**Review by krazziekay — CHANGES_REQUESTED** [review]
Minor changes

**Review by krazziekay — APPROVED** [review]

**Pull request merged**